packages/project-utils/traverseLoaders.js

```diff
@@ -1,16 +0,0 @@
-/**
- * A utility to recursively traverse loaders and execute the "onLoader" callback.
```

Comment:
Can you explain why this file was deleted?

Reply:
It doesn't look like this file was recovered. Can you double check?

BTW this file is not used directly within our `webiny-js` repo, but it is a helper used by Webiny users. In fact, it's mentioned in this doc:  https://www.webiny.com/docs/core-development-concepts/extending-and-customizing/integrate-tailwindcss#header

Reply:
After reverting the change file is no longer in the list of changed files - so it's not deleted anymore.